executor/distsql.go

```diff
@@ -453,6 +458,7 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 func (e *IndexLookUpExecutor) startTableWorker(ctx context.Context, workCh <-chan *lookupTableTask) {
 	lookupConcurrencyLimit := e.ctx.GetSessionVars().IndexLookupConcurrency
 	e.tblWorkerWg.Add(lookupConcurrencyLimit)
+	e.baseExecutor.ctx.GetSessionVars().StmtCtx.RuntimeStats.GetRuntimeStat(e.id + "_tableReader")
```

Comment:
I think this collected stats is never used by the `ExplainExec`

Reply:
`e.id + "_tableReader"` question~~?

Reply:
Because you only retrieve the runtime statistics of an operator by the operator id.

Reply:
https://github.com/pingcap/tidb/pull/7827/files#diff-31b1cb91591c88f55f52073d912b1e82R455

Reply:
oh, this is by design, `GetRuntimeStat` will pre-init `e.id + "_tableReader"` slot in RuntimeStatsColl,  after this call will fork goroutine create executors, this pre-init prevent race condition for modify RuntimeStatsColl map, so `RuntimeStatsColl` live without mutex.

Reply:
I think we don't need to collect the runtime statistics of the table worker of the IndexLookupExecutor, because it is never used and presented to the user in the `ExplainExec`.

Reply:
ok, `_tableReader` can be removed~